src/index.html

```diff
@@ -7,14 +7,25 @@
     <title>Ooyala Receiver</title>
 
     <script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
-    <script type="text/javascript" src="//player.ooyala.com/core/<%= htmlWebpackPlugin.options.playerId %>?plugins=bm"></script>    
+    <script type="text/javascript"
+            src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm,cast"></script>
```

Comment:
it feels like `htmlWebpackPlugin.options.playerId` should be there to be able to modify playerBrandingId from the config.